lib/status_event_repository.rb

```diff
@@ -1,5 +1,5 @@
 require "semantic_logger"
-
+require_relative "bag_status"
```

Comment:
Nit-pick: I like to organize my required/imported libraries into three groups (standard libraries, third-party libraries, local libraries, in that order, alphabetized within each) and keep them separated by newlines.

```suggestion

require_relative "bag_status"
```